### PR TITLE
Stop indexing pages

### PR DESCRIPTION
**Pull request by s3tezsky**
| Q             | A
| ------------- | ---
|Description, reason for the PR| For demo we do not want to index pages by search engines
|New feature| No
|Fixes issues| ...
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: src/Shopsys/ShopBundle/Resources/views/Front/Layout/base.html.twig [hunk 1]
```diff
@@ -8,7 +8,9 @@
         <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
         <meta name="description" content="{% block meta_description %}{% endblock %}">
         <meta http-equiv="X-UA-Compatible" content="IE=11" />
-        {% block meta_robots %}{% endblock %}
+        {% block meta_robots %}
+        <meta name="robots" content="noindex, nofollow">
```
Comment (TomasLudvik) [inline src/Shopsys/ShopBundle/Resources/views/Front/Layout/base.html.twig]: Please add indentation here

**Review by TomasLudvik — CHANGES_REQUESTED** [review]
Hello @s3tezsky,

nice job here. I was thinking if it is ok to add "nofollow" because it was not part of the US, but it makes sense to me, you will just need to explain this to @LukasHeinz during BV. 

There are two small things I want to ask you: 
- in `src/Shopsys/ShopBundle/Resources/views/Front/Inline/Paginator/paginator.html.twig` is now unused macro `metaRobots` please remove this as well.
- squash those two commits with description like "disallow robots indexing of demoshop"

**Comment by s3tezsky** [comment]
I have fixed your comments @TomasLudvik 

> A Noindex tag tells search engines that the relevant page should not be indexed in their search results. If you haven't added a Nofollow to the tag, however, search engines will still follow all the links on the page. In other words, the page will be read by search engines but it won't be indexed.

**Review by TomasLudvik — APPROVED** [review]
CR @ tested on http://ds-no-indexing.loki.shopsys.com. Good job!

**Review by LukasHeinz — APPROVED** [review]

**Pull request merged**